soe/python/manifest.yml

```diff
@@ -13,6 +13,6 @@ applications:
     CONVERSATION_PASSWORD: #CONVERSATION PASSWORD
     CONVERSATION_WORKSPACE_ID: #WORKSPACE ID
     CONVERSATION_VERSION: '2016-07-11'
-# URL for cloudant should look like https://3f99fb1a-f7b9-4007-929d-3d4dcd810d0b-bluemix:2cb3a3b43950fd9c28d3bca97a1ed052456b75794363caced80f566ee43fc582@3f99fb1a-f7b9-4007-929d-3d4dcd810d0b-bluemix.cloudant.com
+# URL for cloudant should look like https://{credentials}:2cb3a3b43950fd9c28d3bca97a1ed052456b75794363caced80f566ee43fc582@3f99fb1a-f7b9-4007-929d-3d4dcd810d0b-bluemix.cloudant.com
```

Comment:
Might be better for it to be 
```
 # URL for cloudant should look like https://{cloudant_account}:{password}@{cloudant_account}-bluemix.cloudant.com 
```
"2cb3a3b43950fd9c28d3bca97a1ed052456b75794363caced80f566ee43fc582" is still some kind of password

and `3f99fb1a-f7b9-4007-929d-3d4dcd810d0b` in the url is also the cloudant username

Reply:
Thanks @jfmartinez, I missed that.